### PR TITLE
[layers] `sparseCategoricalCrossentropy` should have `fromLogits` as a function parameter

### PR DESCRIPTION
**Pull request by BayanBennett**
tfjs-layers/losses:
`sparseCategoricalCrossentropy` should have `fromLogits` as a function param, as specified in the jsdoc comment

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/3156)

### CODE REVIEW

**Comment by fawazahmed0** [comment]
Hey, how do you set the `fromLogits` for `sparseCategoricalCrossentropy` loss function , I would like to do that.
The [API doc](https://js.tensorflow.org/api/latest/) doesn't seem to provide any reference to this loss function though.

Thanks

**Comment by BayanBennett** [comment]
I imported it directly from `tfjs-layers`

```javascript
import { sparseCategoricalCrossentropy } from "@tensorflow/tfjs-layers/dist/losses";
```

**Comment by fawazahmed0** [comment]
> 
> 
> I imported it directly from `tfjs-layers`
> 
> ```js
> import { sparseCategoricalCrossentropy } from "@tensorflow/tfjs-layers/dist/losses";
> ```

Thanks @BayanBennett
